io.thenexusavenger.Nexus-LU-Launcher.yml

```diff
@@ -0,0 +1,52 @@
+app-id: io.thenexusavenger.Nexus-LU-Launcher
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+base: org.winehq.Wine
+base-version: stable-23.08
+sdk: org.freedesktop.Sdk
+command: Nexus-LU-Launcher
+
+finish-args:
+  - --allow=multiarch
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio # Required for audio in LEGO Universe.
+  - --device=dri # Required for OpenGL rendering in LEGO Universe.
+  - --share=ipc # Required for X11.
+  - --share=network # LEGO Universe and patches in launcher require network access. 
+  - --filesystem=home:ro # User-provided client archives need to be readable.
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet8
+add-extensions:
+  org.freedesktop.Platform.Compat.i386:
+    directory: lib/i386-linux-gnu
+    version: '23.08'
+inherit-extensions:
+  - org.freedesktop.Platform.GL32
+  - org.freedesktop.Platform.ffmpeg-full
+  - org.freedesktop.Platform.ffmpeg_full.i386
+  - org.winehq.Wine.gecko
+  - org.winehq.Wine.mono
+  - org.winehq.Wine.DLLs
+
+build-options:
+  append-path: /usr/lib/sdk/dotnet8/bin
+  append-ld-library-path: /usr/lib/sdk/dotnet8/lib
+  env:
+    PKG_CONFIG_PATH: /app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/dotnet8/lib/pkgconfig
```

Comment:
```suggestion
  append-ld-library-path: /usr/lib/sdk/dotnet8/lib
  append-pkg-config-path: /usr/lib/sdk/dotnet8/lib/pkgconfig
```

Reply:
Change made. No issues locally with this change.